### PR TITLE
feat: add node secure protocol options.

### PR DESCRIPTION
**Pull request by KagChi**

### CODE REVIEW

**Review by Deivu — CHANGES_REQUESTED** [review]
Add the default value of secure on ShoukakuConstants, then move secure above the group?: in typings file, other than that, just add the missing docs and its good to go

**Comment by KagChi** [comment]
is this correct?

**Comment by Allvaa** [comment]
i think the option should be optional

**Comment by Deivu** [comment]
I agree, since it gets merged to default constants value anyways, it could be optional instead as @Allvaa recommended

File: src/constants/ShoukakuConstants.js [hunk 1]
```diff
@@ -109,6 +109,7 @@ class ShoukakuConstants {
     * @property {string} host Your node host / ip address of where the lavalink is hosted.
     * @property {number} port The Port Number of your lavalink instance.
     * @property {string} auth The authentication key you set on your lavalink config.
+    * @property {boolean} secure the protocols should secure or no.
```
Comment (Deivu) [inline src/constants/ShoukakuConstants.js:112]: just wrap secure with [] and ? on start of boolean to signify its optional on docs

**Review by Deivu — APPROVED** [review]

**Comment by Deivu** [comment]
Seems good merging this thanks for the PR

**Pull request merged**